src/OowpQuery.php

```diff
@@ -25,10 +25,15 @@ function __construct($query = '')
         );
         $query    = wp_parse_args($query, $defaults);
 
-        // if there is no post type, or the post type is singular and isn't valid, replace it with 'any'
+        // If there is no post type, or the post type is singular and isn't valid, replace it with any *except*
+        // 'attachment' which can cause crashes on ?preview=true if a file title matches a render-able post's.
         if (!isset($query['post_type']) || (!is_array($query['post_type']) && !array_key_exists($query['post_type'],
                     $wp_post_types))) {
-            $query['post_type'] = 'any';
+            $query['post_type'] = array_values(get_post_types());
```

Comment:
`$query['post_type'] = array_values(array_diff(get_post_types(), ['attachment']));` would do the same thing, but a little easier to read (imo)

Reply:
Good call, thanks @rasmuswinter. I've used long array syntax as we seem to use it elsewhere and don't explicitly state a minimum PHP version, so wasn't sure if short arrays would risk a new BC break